local ServerScriptService = game:GetService("ServerScriptService")

local PlayerService     = require(ServerScriptService.Modules.PlayerService)
local EconomyService    = require(ServerScriptService.Modules.EconomyService)
local LuckService       = require(ServerScriptService.Modules.LuckService)
local RouletteService   = require(ServerScriptService.Modules.RouletteService)
local PetService        = require(ServerScriptService.Modules.PetService)
local BalloonService    = require(ServerScriptService.Modules.BalloonService)
local BaseService       = require(ServerScriptService.Modules.BaseService)
local RebirthService    = require(ServerScriptService.Modules.RebirthService)
local IndexService      = require(ServerScriptService.Modules.IndexService)
local PendingPetService = require(ServerScriptService.Modules.PendingPetService)
local PetMovementService = require(ServerScriptService.Modules.PetMovementService)

PlayerService.Init()
EconomyService.Init()
LuckService.Init()
RouletteService.Init()
PetService.Init()
BalloonService.Init()
BaseService.Init()
RebirthService.Init()
IndexService.Init()
PendingPetService.Init()
PetMovementService.Init()

PlayerService.Start()
EconomyService.Start()
LuckService.Start()
RouletteService.Start()
PetService.Start()
BalloonService.Start()
BaseService.Start()
RebirthService.Start()
IndexService.Start()
PendingPetService.Start()
PetMovementService.Start()